cmd/sonobuoy/app/gen_plugin.go

```diff
@@ -36,7 +36,7 @@ func init() {
 		Args:  cobra.ExactArgs(1),
 	}
 
-	GenCommand.PersistentFlags().StringArrayVarP(
+	cmd.PersistentFlags().StringArrayVarP(
```

Comment:
this appears to be a stray change, should it be in this commit? (totally cool if it's a bug fix or whatever)

Reply:
yeah it's a bug fix I noticed while I was fixing this. The plugin path gets added to the parent command by accident.